cypress/e2e/PAGES.ts

```diff
@@ -0,0 +1,12 @@
+interface Page {
+    friendly_name: string;
+    url: string;
+}
+
+const PAGES: Page[] = [
+    { friendly_name: "Clients", url: "/clients" },
+    { friendly_name: "Calendar", url: "/calendar" },
+    { friendly_name: "Lists", url: "/lists" },
```

Comment:
Do other pages such as add-clients need to be added? Or is that for us to do as we make the pages?

Reply:
both - if you make a new page then you should add to PAGES to add to all tests, but at the moment these are the only pages on main!